inc/osvr/ClientKit/Imaging.h

```diff
@@ -94,29 +92,25 @@ namespace clientkit {
                                      const struct OSVR_ImagingReport *report) {
                 ImagingCallbackRegistration *self =
                     static_cast<ImagingCallbackRegistration *>(userdata);
-                ImagingReportOpenCV newReport;
+                ImagingReport newReport;
                 newReport.sensor = report->sensor;
                 newReport.buffer.reset(report->state.data,
                                        ImagingDeleter(self->m_ctx));
-                newReport.frame = cv::Mat(
```

Comment:
But I thought we didn't want to encourage people to downgrade to the C interface just to get rid of a dependency? See issue #44.